woodchipper/logger.py

```diff
@@ -7,8 +7,14 @@
 
 
 class BoundLogger(StdlibBoundLogger):
+    key_prefix: str = ""
```

Comment:
If we're going to declare this as an instance attribute, let's give it a dunder name, so that it's a private attribute.